node/src/GlideClusterClient.ts

```diff
@@ -567,4 +568,25 @@ export class GlideClusterClient extends BaseClient {
     public time(route?: Routes): Promise<ClusterResponse<[string, string]>> {
         return this.createWritePromise(createTime(), toProtobufRoute(route));
     }
+
+    /**
+     * Returns the number of keys in the database.
+     *
+     * See https://valkey.io/commands/dbsize/ for more details.
+
+     * @param route - The command will be routed to all primaries, unless `route` is provided, in which
+     *   case the client will route the command to the nodes defined by `route`.
+     * 
+     * @returns The number of keys in the database.
+     *     If the query is routed to multiple nodes, returns the sum of the number of keys across all routed nodes.
```

Comment:
```suggestion
     *     In the case of routing the query to multiple nodes, returns the aggregated number of keys across the different nodes.
```
lets have a similar doc in all wrappers 🙂